src/containers/Auth/Login/Login.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+import * as actions from '../../../store/actions/index'
+
+import '../../../assets/semantic/semantic.min.css';
```

Comment:
remove `semantic.min.css` from all files this should import on src/index.js only